Update All project file is added

ozovalihasan opened the pull request:
I worked alone.
Grid and flex are used.

ozovalihasan committed:
Add All starting configurations is done.

The repo is cloned.
linter settings are done.
reset.css is added.
main.css and index.html are created.
.gitignore file is created and filled.
README file is filled.
Repository is enabled to be used by stickler-ci.com .

ozovalihasan committed:
Fix Author links in readme file are fixed.

ozovalihasan committed:
Add index.html is started with empty hmtl file.

ozovalihasan committed:
Milestone All files of first draft of project is added.

Background images are added.
Buttons are added.
Footer is added.
All texts are added.
All elements are aligned.
Hover effect of buttons is applied.

ozovalihasan committed:
Add Animation for search button in nav-bar is added

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Fix Search button animation is fixed. Screenshot of project is updated.

ozovalihasan committed:
Update Live demo link is updated.

ozovalihasan committed:
Fix Errors found by linters are fixed.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Update Live demo link is updated.

MarioDena reviewed (changes requested):
Hello! Good work on your project! 👍  however there are some changes that need to be made.

## Changes required

- [ ] Please remove the pink background around the images at the bottom.

- [ ] Please fix the background image as it appears incorrectly.

![img](https://raw.githubusercontent.com/MarioDena/reviewImages/master/apr2020/img2.png)

- [ ] There are some repeated rules on your CSS please remember to keep your code DRY (don't repeat yourself) for example `display: flex;` is repeated 5 times. Article for support: https://speckyboy.com/good-bad-css-practices/

## Tips

- [ ] Test your project against several resolutions, to test your sites on higher resolution than your own use the developer tools on your browser.

After making the requested changes please resubmit your project for another review. 

Cheers! 💯

ozovalihasan committed:
Milestone Allmost all changes given by feedback is applied.

Background color of bottom images is deleted.
Main background image is adjusted as covering necessary part of page.
Page width and height is adjusted as covering all viewport for higher resolution screens.

ozovalihasan committed:
Update Codes in css file is reduced by using DRY approach. Width and height are adjusted for high resolurion screen

ozovalihasan committed:
Fix Errors found by linters are fixed.

ozovalihasan committed:
Update Page is update to be suitable for high resolution secreen.

ozovalihasan committed:
Fix Errors found by linters is fixed.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Update Live demo link is updated.

ozovalihasan committed:
Update Necessary adaptations to change size of buttons with changing screen resolution are done

ozovalihasan committed:
Update Live demo link is changed.

ozovalihasan committed:
Update Live demo link is updated.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Fix Size of button are fixed.

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 reviewed (approved):
## Status: Approved 🏆 🎈 

Hi @ozovalihasan, you have done a great job implementing the changes requested by the previous TSE 👍 I however, approve of your project. Before we merge to master, let us add our email to the author part of the README file.
Go ahead and merge to master 🤝 
_Good luck with your next projects_ 😄 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

ozovalihasan committed:
Fix Size of button are fixed.

ozovalihasan committed:
Update Live demo link is updated.

ozovalihasan committed:
Update Screenshot of project is updated.

ozovalihasan committed:
Add email is added to readme file
